insights/mappers/systemctl_show.py

```diff
@@ -1,20 +1,74 @@
+"""
+/bin/systemctl show <Services_Name> - Command
+=============================================
+
+This command shows all the paramters for this service, these parameters can be set
+through unit files corresponding to that service or from command line.
+
+Typical output of ``/bin/systemctl show mariadb`` command is::
```

Comment:
One last comment: 
Would it be better to shift each "Typical output" and "Examples" to each corresponding class?